src/icalendar/attr.py

```diff
@@ -97,8 +97,8 @@ def _get_rdates(
     """
     result = []
     rdates = self.get("RDATE", [])
-    for rdates in (rdates,) if not isinstance(rdates, list) else rdates:
-        for dts in rdates.dts:
+    for rdate_item in (rdates,) if not isinstance(rdates, list) else rdates:
```

Comment:
This is an example of fiddly changes. It's not just one file, but the entire PR. I'm sorry, but again, I'm not going to spend time reviewing changes that are not material to its purpose.